lib/platform/github/index.ts

```diff
@@ -145,18 +140,32 @@ async function getBranchProtection(
   return res.body;
 }
 
+export async function getJsonFile(fileName: string): Promise<any | null> {
+  try {
+    return JSON.parse(
+      Buffer.from(
+        (
+          await githubApi.getJson<{ content: string }>(
+            `repos/${config.repository}/contents/${fileName}`
+          )
+        ).body.content,
+        'base64'
+      ).toString()
+    );
+  } catch (err) /* istanbul ignore next */ {
+    return null;
+  }
+}
```

Comment:
here too, move to util / helper, so we can reuse from preset later.

Reply:
Let's do this in subsequent PRs